docs/source/designs/backup-recovery.rst

```diff
@@ -0,0 +1,515 @@
+.. Heading order = - ~ `
+
+ZKAP Database Replication / Recovery
```

Comment:
That's an interesting point.  Considering your suggested connotations, I end up feeling like this system actually does both - it recovers state from the grid (it has to find it using the replica readcap and then download it) and then it uses that state to restores the SQLite3 database it needs in order to operate normally (because the on-grid state is not the SQLite3 database itself, just information which can be used to recreate -- or "reconstruct" it, even).

Separate from that, I think there's a different sense of "recovery" which is slightly more common in software systems - the sense of "failure recovery" or "disaster recovery" where the understanding is that this is a thing you do in order to transition a system in a failed state back to a working state.  I'm not sure how widely this sense would be understood outside of software systems ... but maybe that doesn't matter since this is a software system design. :)

However, these conclusions arise from a detailed knowledge of the implementation strategy chosen.  Given that, maybe the name of the overall system being designed doesn't need to reflect them.  It should reflect the value delivered to the consumer of the system.  I think that semantic distinctions between these words is blurry enough that I have a hard time seeing which is a better fit from that external perspective (perhaps partially because I've been inside it too long).

Since it sounds like you don't think this is critical to understanding of the system I'll probably leave it as-is.